Kimjin/3주차 과제/과제3/style.css

```diff
@@ -0,0 +1,54 @@
+body {
```

Comment:
현재 아래 사진처럼 스크롤이 생기고 있어요. 이것은 body에 기본적으로 있는 `margin` 스타일 때문이에요.
![image](https://user-images.githubusercontent.com/87893624/182092262-f492a53e-f6ec-4b37-97d1-fe07518eeb1c.png)

f12를 눌러서 개발자 도구를 들어가 보면 이렇게 `margin`이 기본적으로 적용되어 있는 것을 볼 수 있어요.
![image](https://user-images.githubusercontent.com/87893624/182092604-0e162a93-dd71-4cdd-8849-c59f88c4e53f.png)

그래서 우리는 항상 body에 `margin: 0`을 줘서 스크롤이 생기지 않도록 해야 합니다😉
```suggestion
body {
    margin: 0;
```